TYPO3.Neos/Classes/TYPO3/Neos/Domain/Strategy/AssetUsageInNodePropertiesStrategy.php

```diff
@@ -89,23 +90,16 @@ public function getUsageReferences(AssetInterface $asset)
             return $this->firstlevelCache[$assetIdentifier];
         }
 
-        $userWorkspace = $this->userService->getPersonalWorkspace();
-
         $relatedNodes = [];
         foreach ($this->getRelatedNodes($asset) as $relatedNodeData) {
-            $accessible = $this->domainUserService->currentUserCanReadWorkspace($relatedNodeData->getWorkspace());
-            if ($accessible) {
-                $context = $this->createContextMatchingNodeData($relatedNodeData);
-            } else {
-                $context = $this->createContentContext($userWorkspace->getName());
-            }
-            $site = $context->getCurrentSite();
+            $context = $this->createContextMatchingNodeData($relatedNodeData);
             $node = $this->nodeFactory->createFromNodeData($relatedNodeData, $context);
             $flowQuery = new FlowQuery([$node]);
-            /** @var \TYPO3\TYPO3CR\Domain\Model\NodeInterface $documentNode */
+            /** @var NodeInterface $documentNode */
             $documentNode = $flowQuery->closest('[instanceof TYPO3.Neos:Document]')->get(0);
-
-            $relatedNodes[] = new AssetUsageInNodeProperties($asset, $site, $documentNode, $node, $accessible);
+            $site = $context->getCurrentSite();
+            $accessible = $this->domainUserService->currentUserCanReadWorkspace($relatedNodeData->getWorkspace();
```

Comment:
In the issue I had, the workspace of the node wasn't accessible. Because of that the code tried to fetch the node from the user's workspace, but the node didn't exist there.

I don't think it is possible to have access to the workspace, but not to a node in that workspace. I think you thought of a node shining through from another workspace (which perhaps is not accessible by the user [probably don't possible either]), but then the node's workspace would be another...